<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proxyui</class>
 <widget class="QMainWindow" name="proxyui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>proxyui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>691</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lbl_port">
         <property name="text">
          <string> Digite a porta TCP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_port">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pb_start">
       <property name="text">
        <string>Iniciar servidor proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="pb_request">
           <property name="text">
            <string>Request</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="te_request">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pb_response">
           <property name="text">
            <string>Response</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="te_response">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_spider">
         <property name="text">
          <string>URL:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_spider"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pb_spidergo">
       <property name="text">
        <string>Spider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="te_spider">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lbl_dump">
         <property name="text">
          <string>URL:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_dump"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pb_dump">
       <property name="text">
        <string>Dump</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServidor_Proxy">
    <property name="title">
     <string>Servidor_Proxy</string>
    </property>
   </widget>
   <addaction name="menuServidor_Proxy"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>

